js/packages/phoenix-client/src/experiments/runExperiment.ts

```diff
@@ -197,6 +202,26 @@ export async function runExperiment({
     );
   }
 
+  if (!isDryRun && client.config.baseUrl) {
+    const datasetUrl = getDatasetUrl({
+      baseUrl: client.config.baseUrl,
+      datasetId: dataset.id,
+    });
+    const datasetExperimentsUrl = getDatasetExperimentsUrl({
+      baseUrl: client.config.baseUrl,
+      datasetId: dataset.id,
+    });
+    const experimentUrl = getExperimentUrl({
+      baseUrl: client.config.baseUrl,
+      datasetId: dataset.id,
+      experimentId: experiment.id,
+    });
+
+    logger.info(`📊 View dataset: ${datasetUrl}`);
```

Comment:
nice! We should probably add these also to the dataset methods - though I wonder if we should have a no-op logger. Not blocking